src/components/Picker/docs/PickerWiringExample.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Picker } from '../';
+import styles from './PickerWiringExample.st.css';
+
+export const PickerWiringExample = () => {
+  return (
+    <Picker
+      value={'October 2020'}
+      previousClickHandler={() => {}}
+      nextClickHandler={() => {}}
+      {...styles('root', {})}
+    />
+  );
+};
```

Comment:
Let's have a bit more options to this example.
For instance add a state for the value, so that we'll have some options.
ie:
```suggestion
const months = ['January', 'February', ... , 'December'];
export const PickerWiringExample = () => {
  const [value, setValue] = React.useState(months.length / 2);
  return (
    <Picker
      value={'October 2020'}
      onPrev={() => setValue(Math.max(value - 1, 0))}
      onNext={() => setValue(Math.min(value + 1, months.length - 1))}
      disableNext={value === months.length - 1}
      disablePrev={value === 0}
      {...styles('root', {})}
    />
  );
};
```